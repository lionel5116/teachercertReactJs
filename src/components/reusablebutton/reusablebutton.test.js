import React from 'react';
import ReactDOM from 'react-dom';
import { create } from "react-test-renderer";
import Reusablebutton from './reusablebutton';

it('It should mount', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Reusablebutton />, div);
  ReactDOM.unmountComponentAtNode(div);
});

//npm install --save-dev react-test-renderer
describe("Button component", () => {
  test("Matches the snapshot", () => {
    const button = create(<Reusablebutton />);
    expect(button.toJSON()).toMatchSnapshot();
  });
});

