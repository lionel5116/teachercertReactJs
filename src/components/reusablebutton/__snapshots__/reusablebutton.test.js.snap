// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<div>
  <button
    className="btn btn-primary"
    disabled={false}
    type="button"
  />
</div>
`;
